pkg/p2p/libp2p/internal/handshake/handshake_test.go

```diff
@@ -188,7 +187,7 @@ func TestHandle(t *testing.T) {
 
 	t.Run("OK", func(t *testing.T) {
 		node2Info := Info{
-			Address:   "node2",
+			Address:   []byte("node2"),
```

Comment:
Use swarm.Address not []byte as type, and construct it from actual byteslice or use swarm.NewAddress if it is constructed from hex-encoded string.